p2p-specs/canonical-mempools.md

```diff
@@ -0,0 +1,8 @@
+## The list of published canonical mempools are:
+
+| Chain Name | Chain ID | Mempool ID | Description (Link to Mempool file) |
```

Comment:
none of these files is available. 
so either we push a folder with these files here (so at least users can verify the content with the canonical mempool hash), or pin them, to make them available..

Reply:
It turns out the issue is with Infura, these files were pinned using Infura's account. They are looking to resolve this asap. I shall in the meanwhile push the yaml files as a separate folder as well.

Reply:
> none of these files is available. so either we push a folder with these files here (so at least users can verify the content with the canonical mempool hash), or pin them, to make them available..

This has been done as well.

Reply:
how do you create the CID from the file content? is there a standard tool
(e.g. for md5 you can do `cat file|md5sum` )